src/registrar/utility/errors.py

```diff
@@ -57,6 +57,44 @@ def __str__(self):
         return f"{self.message}"
 
 
+class LoadOrganizationErrorCodes(IntEnum):
```

Comment:
**blocking suggestion** I think this should be moved out of the usual utility, as the data migration code is logically separate from the purpose of the application and these error messages should NOT be used anywhere else, thus we don't want them in a global utility.